### DISCUSSION TITLE
I want to add a horizontal menu for subpages to an entity. How would I do that?

### DISCUSSION THREAD

**Discussion by anxgstadler** [moonshine-software/moonshine#1170] [Q&A]
I have a list of users. When I click on a user I would like to get to a page with the user's name as the headline, and the rest is split into several parts via a horizontal menu:

Part 1) Base data: A form with name, user, address and so on.
Part 2) A list of the user's roles, editable. (This would be a complete ModelRessource on its own, I guess?)
Part 3) A list of contracts the user has, editable. (This also would be a separate ModelRessource, I guess?)
Part 3) A list of purchases the user has done. (This also would be a separate ModelRessource, I guess?)

You get the idea.

So my question would be: How can I create and render such a submenu and how would I go on then linking it to all the ressource index pages (which would have to filtered to the user whose page I have open)?

Thanks for your input!

**Comment by lee-to** [comment]
Make a component with such a menu and implement the logic

**Reply by anxgstadler** [reply]
Thank you, Captain Obvious ;-)

**Reply by lee-to** [reply]
I apologize, but if you expected ready-made code, then of course I will disappoint you
